src/IO/LzmaWriteBuffer.cpp

```diff
@@ -96,26 +92,26 @@ void LzmaWriteBuffer::finish()
 
     next();
 
-    do {
+    do
+    {
         out->nextIfAtEnd();
         lstr.next_out = reinterpret_cast<unsigned char *>(out->position());
         lstr.avail_out = out->buffer().end() - out->position();
 
         lzma_ret ret = lzma_code(&lstr, LZMA_FINISH);
         out->position() = out->buffer().end() - lstr.avail_out;
 
-		//std::cout << ret << " RET FIN" << std::endl;
 
-        if (ret == LZMA_STREAM_END) {
+        if (ret == LZMA_STREAM_END) 
```

Comment:
trailing whitespace, and incorrect indent below (tabs instead of spaces, and tabs also not only here)

And by the way it is not only here, it is pretty easy to fix just using git, but please note that this will override the history (and that said that if you don't know git I guess it will be better if you will fix this in your editor)

```
git rebase --whitespace=fix upstream/master
```

Reply:
Can I just format it with `clang-format` and options from .clang-format?

Reply:
I guess (I've never used it)